src/lib/stitching/gravity/stitching.ts

```diff
@@ -90,7 +103,9 @@ export const gravityStitchingEnvironment = (
           before: String
         ): UserAddressConnection
       }
-
+      extend type UserAddress implements Node {
```

Comment:
why `implements Node`? are you planning on using the node resolver on this (`node(id: ...)`). If not, this should be removed. Additionally this shouldn't be stitched into V1 if it can be avoided.

Reply:
Thanks Matt, The reason I have added the `id` was to be able to find the node in the relay store and update after mutation (from my understanding that was needed). Otherwise, I don't need something like `node(id:..` so I guess I can ✂️ that.

Reply:
@mzikherman I got rid of `Node` and the v1/v2 conditionals. Does the PR look good to be merged?